src/pages/guidelines/index.astro

```diff
@@ -42,18 +42,11 @@ import GuidelinesRespec from "@/components/respec/GuidelinesRespec.astro";
 					<li>We would like feedback on this draft. You can raise a <a href="https://github.com/w3c/wcag3/issues">GitHub issue</a> or email <a href="mailto:public-agwg-comments@w3.org?subject=WCAG%203.0%20public%20comment">public-agwg-comments@w3.org</a>.</li>					
 				</ul>
 			</details>
-			<h3>What's new in this version of WCAG 3.0?</h3>
+			<h3>About this draft</h3>
```

Comment:
I think the lead "What's new..." is more eye catching and provides better clarity on the information in the section.

What about:
```suggestion
			<h3>What's new in this draft?</h3>
```

Reply:
While I agree it's more catching, it's not true. This section doesn't really explain what's new. It only says, "This draft includes an updated list of the potential guidelines, requirements, and assertions that have progressed to Developing status." It would be helpful if it listed which were updated, yet I assume there are reasons not to do that?

The section is mostly review questions and how to comment. Then sub-sections are about the 'Draft requirements' and 'Section status levels'.

Reply:
humm...

/me skims https://www.w3.org/TR/2024/WD-wcag-3.0-20241212/ and sees lots at Developing

Use case: I reviewed the last publication, and I want to know what's changed since then...